packages/dappmanager/hostScripts/docker_update.sh

```diff
@@ -0,0 +1,343 @@
+#!/bin/bash
```

Comment:
I strongly believe this script can be simplified a lot by splitting the docker and compose update logic in separate scripts. Then you will not need to parse arguments so much and handle branching logic